### PR TITLE
[ru] translate articles of osu! namespace

### PR DESCRIPTION
**Pull request by TicClick**

### CODE REVIEW

**Comment by MegaApplePi** [comment]
just noting, know that the ASG (Article Style Guide) is currently subject to change (see #155)

**Comment by sr229** [comment]
Based from in depth:

- Your PR has no issues with formatting (Inclined with ASG, Good Job!)


Again, you would need this to be reviewed properly by someone else that extensively knows Russian for grammar-checking.

**Comment by TicClick** [comment]
yeah I figured

File: wiki/Article_Style_Guide/ru.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+**Не** используйте для этой цели верхний регистр.
+Заменяйте его **жирным** начертанием или *курсивом* (или ***жирным курсивом***). Пример:
+
+- Что бы ни случилось, ***всегда*** переходите дорогу на красный свет.
+
+## Участки кода
+
+Оформляйте как код:
+
+- Название клавиш: `B`, `Ctrl` + `Shift` + `A` (клавиши-модификаторы пишутся с большой буквы);
```
Comment (TicClick) [inline wiki/Article_Style_Guide/ru.md]: таки да, я потом asg ещё целиком пересмотрю, раз он обновляется, как мне наверху подсказали

File: wiki/Article_Style_Guide/ru.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+
+### Карты (мапсеты)
+
+Ссылка на карту должна иметь следующий формат:
+
+```
+{исполнитель} - {название} ({маппер})
+```
+
+Ссылка на мапсет имеет вид `osu.ppy.sh/b/{ID мапсета}`.
```
Comment (TicClick) [inline wiki/Article_Style_Guide/ru.md]: ой!

File: wiki/How_You_Can_Help!/ru.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+Заводите обсуждения, если не уверены в правках, или присылайте пулл-реквесты с обновлениями.
+- Следите за [пулл-реквестами](https://github.com/ppy/osu-wiki/pulls "Пулл-реквесты") (можно даже в закладки добавить), смотрите, чтобы никто не добавил глупостей или ошибок, пишите к ним ревью.
+Будьте в теме, насколько возможно!
+- Следите, чтобы информация на вики была как можно более точной, ёмкой и понятной, как в смысле содержания, так и форматирования.
+- Да хотя бы переведите эту статью *(прим. ред.: уже)*!
+Нам нужна помощь всего сообщества, не только его англоговорящей части (занимательный факт: более 75% пользователей плохо понимают английский или не знают его совсем).
+
+## Поддерживайте osu! в чистоте!
+
+В osu! есть модераторы, но за потоком активности в 25+ каналах — и тем более на форумах — трудно пристально следить, но это легко исправить.
```
Comment (TicClick) [inline wiki/How_You_Can_Help!/ru.md]: да-да

File: wiki/How_You_Can_Help!/ru.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+### Откуда начать:
+
+- Читайте свои любимые чаты и форумы.
+- Если видите что-то неподобаюющее или нарушающее правила, используйте команду `!report`, чтобы связаться с модераторами.
```
Comment (TicClick) [inline wiki/How_You_Can_Help!/ru.md]: right on

File: wiki/How_You_Can_Help!/ru.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+- Выберите один из этих форумов: [Техподдержка](http://osu.ppy.sh/forum/5), [Помощь с маппингом](http://osu.ppy.sh/forum/56), [Сторибординг](http://osu.ppy.sh/forum/20), [Геймплей и ранкинги](http://osu.ppy.sh/forum/13).
+- Ткните в ссылку Mark as read в правом верхнем углу.
+- Следите за новыми темами (они помечены яркими иконками) и по возможности туда отвечайте.
+- Читайте [руководства на вики](/wiki/List_of_Guides "Список руководств") и просвещайтесь.
+Допишите их, если чувствуете в себе силы.
+
+## Проверяйте и модьте карты!
+
+Несмотря на то, что для ранкинга существуют отдельные команды, помогать может любой желающий.
```
Comment (TicClick) [inline wiki/How_You_Can_Help!/ru.md]: да

File: wiki/How_You_Can_Help!/ru.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+Пишите отзывы к опубликованным картам, оставляйте конструктивную критику, и, возможно, когда-нибудь вы попадёте в одну из тех самых команд!
```
Comment (TicClick) [inline wiki/How_You_Can_Help!/ru.md]: угуу

File: wiki/How_You_Can_Help!/ru.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+### Откуда начать:
+
+- Загляните в [турнирный форум](http://osu.ppy.sh/forum/55).
+Возможно, вы придётесь кстати, например, выступите в качестве зрителя или судьи.
+- Исследуйте [форум по разработке](http://osu.ppy.sh/forum/2).
+Активность там низкая, но [проекты](http://osu.ppy.sh/forum/t/58855) [более](http://osu.ppy.sh/forum/t/80027) [серьёзны](http://osu.ppy.sh/forum/t/79354) и масштабны.
+- Влейтесь в какой-нибудь [проект по маппингу](http://osu.ppy.sh/forum/53) и замутите с другими мапперами коллаб.
+- Если у вас есть классная идея — начните свой проект!
+Если требуется помощь или совет, можете попробовать написать мне (тут есть шанс, что я не отвечу, если у меня много дел).
+- Оживите [osu!monthly](http://osu.ppy.sh/forum/t/75017), я без него скучаю :<
```
Comment (TicClick) [inline wiki/How_You_Can_Help!/ru.md]: [Тёмыч говорит, что обычно нет](https://www.artlebedev.ru/kovodstvo/sections/143/#17), так что!

**Comment by MegaApplePi** [comment]
is this ready to merge?

**Comment by TicClick** [comment]
why, yes

**Comment by MegaApplePi** [comment]
oh, alrighty 👍 we usually tell Repflez on the osu!dev discord chat when pull requests are ready to be merged

**Comment by TicClick** [comment]
ugh, didn't know that. shoulda probably joined it

**Comment by MegaApplePi** [comment]
probably... it is _quickest_ (though, sometimes not the fastest) way to get feedback.. and the best way to ping for a merge

**Pull request merged**